hassio/src/addon-view/hassio-addon-info.ts

```diff
@@ -428,6 +433,7 @@ class HassioAddonInfo extends LitElement {
                         tabindex="-1"
                         target="_blank"
                         class="right"
+                        rel="noopener"
```

Comment:
@frenck should this also be `noreferrer` or Paulus, why is this one `noopener`? Because it is a link to user owned location?

Reply:
`noopener` prevents javascript in the new window to access this current window. Which makes sense, since it would be a third party application running here, right?

Reply:
This is a local link actually. Just doing noopener so that it can't hurt our performance, there wasn't really a need for no referrer. 

No referrer = no opener + no referrer header.